internal/cmd/group_flag.go

```diff
@@ -190,10 +190,38 @@ func countFlags(flags ...string) (count int) {
 }
 
 const MaxAWSDBSizeBytes = 1024 * 1024 * 1024 * 20 // 20 GB
+func checkIfDump(filename string) (bool, error) {
```

Comment:
Should we instead check that file is SQLite db (by simply checking sqlite magic header string)?

Reply:
We can do that, certainly. But if it *isn't* a sqlite db, then what is it? The message "this is a dump" is more helpful to users than "this is not a valid database". An incredible amount of people confuse the two!